state/execution.go

```diff
@@ -551,18 +551,36 @@ func fireEvents(
 // ExecCommitBlock executes and commits a block on the proxyApp without validating or mutating the state.
 // It returns the application root hash (result of abci.Commit).
 func ExecCommitBlock(
+	be *BlockExecutor,
 	appConnConsensus proxy.AppConnConsensus,
 	block *types.Block,
 	logger log.Logger,
 	store Store,
 	initialHeight int64,
+	s State,
```

Comment:
Where is this used?

Reply:
at line 570, the 'ConsensusParams.Validator'

Reply:
Right, I couldn't see it